netdiff/parsers/openvpn.py

```diff
@@ -16,8 +17,12 @@ class OpenvpnParser(BaseParser):
     def to_python(self, data):
         try:
             return parse_status(data)
-        except (AttributeError, ParsingError):
-            return None
+        except (AttributeError, ParsingError) as e:
+            msg = 'something went wrong: {0}'.format(str(e))
+            try:
```

Comment:
why you can't just remove this `try` block with the following?

```python
raise ConversionException(msg)
```

Reply:
because another exception is being raised.